core/client/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -63,6 +64,8 @@
     mFileSystemMasterClientPool = new FileSystemMasterClientPool(ClientContext.getMasterAddress());
     mBlockStoreContext = BlockStoreContext.get();
     mAlluxioBlockStore = new AlluxioBlockStore(mBlockStoreContext);
+
+    MetricsSystem.startSinks();
```

Comment:
Because FileSystemContext is usually the class used by client. Do you have any other suggestion?